UnitsNet/UnitDefinitions/Length.json

```diff
@@ -168,6 +168,66 @@
           "Abbreviations": [ "микродюйм" ]
         }
       ]
+    },
+    {
+      "SingularName": "PrinterPoint",
+      "PluralName": "PrinterPoints",
+      "FromUnitToBaseFunc": "(x/72.27)*2.54e-2",
```

Comment:
Simplify this function to a constant, like `x * 0.0003514598` if I calculated this right.
Preferably this is a constant found in some trusted source so it is easy to go back and verify it later.

Reply:
Sorry Andreas, but changing this to constants introduces accuracy errors. Keep the formula makes sure this is the most accurate way it can be expressed, without introducing explicitly accuracy errors.

Reply:
I see. The factor has infinite precision, so capping it would probably introduce more accuracy error than necessary. We can keep it as-is.